app/views/assignments/index.html.erb

```diff
@@ -1,49 +1,59 @@
+
+
 <h1><%= @current_office.name %> Kitchen Schedule</h1>
 
-<table>
-<tr>
-  <th>
-    Dates
-  </th>
-  <th>
-    Employees
-  </th>
-</tr>
 
-<% @weeks.each do |week| %>
-  <tr>
-    <td>
-      <%= print_date(week) %>
-    </td>
+<table class="table table-sm table-borderless">
+  <thead class="bg-dark table-dark">
+    <tr>
+      <th>
+        Dates
+      </th>
+      <th>
+        Employees
+      </th>
+
+      <th></th>
+  
+      <% if can_edit_weeks? %>
+        <th></th>
+      <% end %>
```

Comment:
What happens if you remove the empty `th`s? It seems semantically incorrect to include them.

Reply:
It makes it so the header extends the entire length of the table when the edit links are there. Without it it looks like this: 
![table](https://user-images.githubusercontent.com/32042052/33143559-7ffa9a2e-cf77-11e7-908b-414ebe932616.png)
I think that looks ugly but if there's another way to do it I'll fix it.

Reply:
You might be able to style the `tr` instead of the `th`??

I'd also rather have the empty `th`s than have an ugly table header.